vpr/src/place/move_utils.h

```diff
@@ -9,6 +9,8 @@
 /* To turn off incremental bounding box updates, set this to a huge value */
 #define SMALL_NET 4
 
+extern bool f_placer_debug;
```

Comment:
1. Comment.
2. Why not put this in the placement context or placer opts?  We're trying to get away from naked global variables.